api/server/sdk/server.go

```diff
@@ -369,6 +372,32 @@ func (s *Server) restServerSetupHandlers() (*http.ServeMux, error) {
 	if err != nil {
 		return nil, err
 	}
+	err = api.RegisterOpenStorageClusterPairHandlerFromEndpoint(
+		context.Background(),
+		gmux,
+		s.Address(),
+		[]grpc.DialOption{grpc.WithInsecure()})
+	if err != nil {
+		return nil, err
+	}
+
+	err = api.RegisterOpenStorageMigrateHandlerFromEndpoint(
+		context.Background(),
+		gmux,
+		s.Address(),
+		[]grpc.DialOption{grpc.WithInsecure()})
+	if err != nil {
+		return nil, err
+	}
+
+	err = api.RegisterOpenStorageMigrateHandlerFromEndpoint(
```

Comment:
I think this one is not needed since 384 is adding it already.

Reply:
Done.